lunabot_control/src/mpc.cpp

```diff
@@ -213,7 +206,7 @@ MPC::std_dev_(std::vector<std::pair<Eigen::MatrixXd, double>> rollouts) {
                              (rollouts[i].first(j, 4) - temp_mean(j, 1));
         }
     }
-    // ROS_ASSERT(this->top_rollouts_ > 1);
```

Comment:
Why remove?

Reply:
honestly do not remember lol

Reply:
seems to be a useful check, will add it back